umap/static/umap/map.css

```diff
@@ -1542,6 +1549,22 @@ a.add-datalayer:hover,
     .umap-popup-large .umap-popup-content {
         width: 300px;
     }
+
+    .attribution-toggle {
+        display: inline-block;
+    }
+    .attribution-container {
+        display: none;
+    }
+    .leaflet-control-attribution:active .attribution-container,
+    .leaflet-control-attribution:hover .attribution-container {
+        display: inline;
```

Comment:
```suggestion
        display: inline;
        background-color: rgba(255, 255, 255);
```

Reply:
`#fff` at this point ?

Reply:
Yup, I kept the declaration from Leaflet but `#fff` looks good to me too.